op-service/txmgr/txmgr.go

```diff
@@ -418,32 +418,42 @@ func (m *SimpleTxManager) queryReceipt(ctx context.Context, txHash common.Hash,
 	return nil
 }
 
-// increaseGasPrice takes the previous transaction & potentially clones then signs it with a higher tip.
-// If the tip + basefee suggested by the network are not greater than the previous values, the same transaction
-// will be returned. If they are greater, this function will ensure that they are at least greater by 15% than
-// the previous transaction's value to ensure that the price bump is large enough.
+// increaseGasPrice takes the previous transaction, clones it, and returns it with fee values that
+// are at least 15% higher than the previous ones to satisfy Geth's replacement rules, and no lower
+// than the values returned by the fee suggestion algorithm to ensure it doesn't linger in the
+// mempool. Finally to avoid runaway price increases, fees are capped at 5x suggested values.
 //
-// We do not re-estimate the amount of gas used because for some stateful transactions (like output proposals) the
-// act of including the transaction renders the repeat of the transaction invalid.
-//
-// If it encounters an error with creating the new transaction, it will return the old transaction.
+// We do not re-estimate the amount of gas used because for some stateful transactions (like output
+// proposals) the act of including the transaction renders the repeat of the transaction invalid.
 func (m *SimpleTxManager) increaseGasPrice(ctx context.Context, tx *types.Transaction) *types.Transaction {
+	m.l.Info("bumping gas price for tx", "hash", tx.Hash(), "tip", tx.GasTipCap(), "fee", tx.GasFeeCap(), "gaslimit", tx.Gas())
+	var tip, basefee *big.Int
 	tip, basefee, err := m.suggestGasPriceCaps(ctx)
 	if err != nil {
 		m.l.Warn("failed to get suggested gas tip and basefee", "err", err)
-		return tx
+		// In this case we'll use the old tx's tip as the tip suggestion and 0 for base fee so that
+		// we at least get the min geth fee bump.
+		tip = new(big.Int).Set(tx.GasTipCap())
+		basefee = new(big.Int)
 	}
-	gasTipCap, gasFeeCap := updateFees(tx.GasTipCap(), tx.GasFeeCap(), tip, basefee, m.l)
+	bumpedTip, bumpedFee := updateFees(tx.GasTipCap(), tx.GasFeeCap(), tip, basefee, m.l)
 
-	if tx.GasTipCapIntCmp(gasTipCap) == 0 && tx.GasFeeCapIntCmp(gasFeeCap) == 0 {
-		return tx
+	// Make sure increase is at most 5x the suggested values
+	maxTip := new(big.Int).Mul(tip, big.NewInt(5))
```

Comment:
The `big.NewInt(5)` could be a top level `var` with a godoc

Reply:
done